shards/shards.go

```diff
@@ -634,12 +638,15 @@ search:
 
 		select {
 		case work <- next:
-			// We send the max pending priority as we search to keep frontend informed even
-			// if we don't find matches.
-			var r zoekt.SearchResult
 			pending.append(next.priority)
-			r.MaxPendingPriority = pending.max()
-			sender.Send(&r)
+
+			if workCount = (workCount + 1) % 100; workCount == 0 {
+				// We send the max pending priority as we search to keep frontend informed even
+				// if we don't find matches.
+				var r zoekt.SearchResult
+				r.MaxPendingPriority = pending.max()
+				sender.Send(&r)
+			}
 
 			if shard++; shard == len(shards) {
```

Comment:
We already have this shard counter. Could we not use that instead?

Reply:
yep, didn't see the shard counter :-)